src/libUtils/EvmJsonResponse.h

```diff
@@ -77,13 +77,13 @@ struct CallResponse {
   const std::vector<std::shared_ptr<ApplyInstructions>>& Apply() const {
     return m_apply;
   }
-  const std::vector<std::string>& Logs() const { return m_logs; }
+  const std::string Logs() const { return m_logs; }
   const std::string& ExitReason() const { return m_exitReason; }
   uint64_t Gas() const { return m_gasRemaining; }
   const std::string& ReturnedBytes() const { return m_return; }
   bool isSuccess();
   std::vector<std::shared_ptr<ApplyInstructions>> m_apply;
-  std::vector<std::string> m_logs;
+  std::string m_logs;
```

Comment:
I don't understand the reason for this change.  

This is meant to be a separate entry for every LOGx opcode. Why do we merge them into one string?

Reply:
actually the logs recieved are a JSON array, and the output required to be put in the receipt is also a JSON. So, it is just easier to convert a whole string to JSON. Otherwise would have to go element by element and convert to JSON.

Reply:
I'm surprised the logs received is a JSON array, I believe this is a reminiscent of JSONRPC mechanism being used for communication.  If we decide to change it (and we did plan to move to gRPC, unless we also decide to embed it), and it won't be JSONRPC anymore, it will break.

Reply:
ya but it will break either way i.e. vector<string> or string as both expect a JSON now.

Reply:
Well, that's because `EvmResult` in Rust is

```rust
#[derive(serde::Serialize)]
pub struct EvmResult {
    exit_reason: evm::ExitReason,
    return_value: String,
    apply: Vec<DirtyState>,
    logs: Vec<ethereum::Log>,
    remaining_gas: u64,
}
```

And Vec<ethereum::Log> in JSONRPC ends up encoded, as, well, a JSON array :)  it is a coincidence of Vec being encoded that way.

I'd recommend reverting. I guess for the code to convey the meaning is better than just a small convenience of parsing in one place.  Or at least leave a big fat comment about this.

Reply:
Got it, will revert.